examples/helloworld/query.py

```diff
@@ -20,23 +20,45 @@
     # create a new query
     query = (
         Query(env=env).sql(query="SELECT * FROM car_live_data")
-        # .batching_pilicy(1000, 1000)
+        # .batching_policy(1000, 1000)
         .create()
     )
-    query_id = query.metadata()["id"]
+
+    # query header is the colume definitions of query result table
+    # it is a list of name/value pair
+    # for example : [{'name': 'in_use', 'type': 'bool'}, {'name': 'speed', 'type': 'float32'}]
+    query_header = query.header()
+    pprint(f"query with header {query.header()}")
+
+    # query id is the unqie identification, can be used to control query like cancel/delete
```

Comment:
I would suggest moving queryId related sample code after the query result. When I read this for the first time, I got the impression that query ID is required to get the request result.

Reply:
typo :`unqie`

`can be used to control query like cance/deletel`: here the user can simply use `query.cancel` and `query.delete`, in most cases, there is no need for the user to know the query id